Dockerfile

```diff
@@ -6,14 +6,8 @@ RUN apt-get update && apt-get install -qq -y build-essential libpq-dev nodejs --
 
 RUN YARN_VERSION=$(curl -sSL --compressed https://yarnpkg.com/latest-version) \
   set -ex \
-  && for key in \
-    6A010C5166006599AA17F08146C2130DFD2497F5 \
-  ; do \
-    gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
-  done \
   && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
   && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
```

Comment:
I think you can also remove the `.asc` download, as this is the signature that we're no longer verifying.

Reply:
updated.